src/components/MenuItem.js

```diff
@@ -124,6 +123,13 @@ const MenuItem = props => (
                             />
                         </View>
                     )}
+                    {props.selectable && (
+                        <View style={[styles.selectCircle, styles.alignSelfCenter]}>
+                            {props.selected && (
+                                <Icon src={Expensicons.Checkmark} fill={themeColors.iconSuccessFill} />
+                            )}
+                        </View>
+                    )}
```

Comment:
`OptionRow` is much complex in nature and uses a very specific set of props that are meant for Chat options. It would be much hectic to adjust the styling and all that to use it. Instead, MenuItem seems to be a perfect fit. I took the idea from PaymentMethodList.

But yeah, Let me see how can I refactor this.

Reply:
Ok, I was thinking that the pattern we are using here in the UI is very similar to the `OptionsSelector`, but maybe it is different enough that we should not use it (as I think that one also has a search input).

Maybe at a minimum we can make the code I linked a component and then use it in both places? This way when we update our styles in the future we won't have to update it in two places.

Reply:
> Maybe at a minimum we can make the code I linked a component and then use it in both places? This way when we update our styles in the future we won't have to update it in two places.

Already did! made a small component `RadioBox`